W-21544182-doc-update-adding-other-api-assets (#345)
mulesoft/docs-exchange: modules/ROOT/pages/generating-documentation-with-ai.adoc

endif::[]
:imagesdir: ../assets/images/

> Use Einstein generative AI to generate API documentation for REST APIs, SOAP APIs, AsyncAPI, and GraphQL API in Exchange and streamline how your organization manages APIs. By using AI, you can simplify your API documentation processes. Generative AI creates consistent, accurate, and current API documentation. This frees you to focus on strategic tasks and enhances developer satisfaction.

Reviewer: ```suggestion
Use generative AI to generate API documentation for REST, SOAP, AsyncAPI, and GraphQL in Exchange, streamlining how your organization manages APIs. AI simplifies your processes by creating consistent, accurate, and current content, which frees you to focus on strategic tasks and enhances developer satisfaction.
```

Reviewer: SF is now moving away from the Einstein brand. Use "generative AI" instead.

Use Einstein generative AI to generate API documentation for REST APIs, SOAP APIs, AsyncAPI, and GraphQL API in Exchange and streamline how your organization manages APIs. By using AI, you can simplify your API documentation processes. Generative AI creates consistent, accurate, and current API documentation. This frees you to focus on strategic tasks and enhances developer satisfaction.

You can create documentation for single file REST APIs that don't have any documentation. From the asset details page in Exchange, generate the documentation, then manually edit and improve the documentation. After that, publish it to make it available to everyone in your organization.
You can create documentation for API assets that don't have any documentation. From the asset details page in Exchange, generate the documentation, then manually edit and improve the documentation. After that, publish it to make it available to everyone in your organization.

This release supports:

* REST APIs

> * Single file format

Reviewer: I would leave this in

+

> Multi-format files and fragments aren’t supported.

Reviewer: If mult-format files and fragments are supported, include it as well.

Author: I have added them in line 18 - Generate documentation for these APIs whether they are created using a single specification file or multiple specification files.
Isn't that ok for now? Will check with the team about fragments.
